docs/source/es/index.mdx

```diff
@@ -34,9 +34,9 @@ Cada arquitectura de 🤗 Transformers se define en un módulo de Python indepen
 
 La documentación está organizada en cinco partes:
 
-- **INICIANDO** contiene un recorrido rápido e instrucciones de instalación para comenzar a usar 🤗 Transformers.
+- **EMPEZAR** contiene un recorrido rápido e instrucciones de instalación para comenzar a usar 🤗 Transformers.
 - **TUTORIALES** son un excelente lugar para comenzar si eres nuevo en nuestra biblioteca. Esta sección te ayudará a obtener las habilidades básicas que necesitas para comenzar a usar 🤗 Transformers.
-- **GUÍAS HOW-TO** te mostrará cómo lograr un objetivo específico, cómo hacer fine-tuning a un modelo preentrenado para el modelado de lenguaje o cómo crear un cabezal para un modelo personalizado.
+- **GUÍAS PRÁCTICAS** te mostrará cómo lograr un objetivo específico, cómo hacer fine-tuning a un modelo preentrenado para el modelado de lenguaje o cómo crear un cabezal para un modelo personalizado.
 - **GUÍAS CONCEPTUALES** proporciona más discusión y explicación de los conceptos e ideas subyacentes detrás de los modelos, las tareas y la filosofía de diseño de 🤗 Transformers. 
 - **API** describe cada clase y función, agrupadas en:
```

Comment:
I don't think there is any "API" section at the moment, we can also remove this I think